tensorboard/plugins/distributions/tf_distribution_dashboard/tf-distribution-loader.html

```diff
@@ -0,0 +1,162 @@
+        icon="fullscreen"
+        on-tap="_toggleExpanded"
+      ></paper-icon-button>
+    </div>
+    <style>
+      :host {
+        display: flex;
+        flex-direction: column;
+        width: 330px;
+        height: 235px;
+        margin-right: 10px;
+        margin-bottom: 15px;
+      }
+      :host[_expanded] {
+        width: 700px;
+        height: 500px;
+      }
+
+      vz-histogram-timeseries {
+        -moz-user-select: none;
+        -webkit-user-select: none;
+      }
+
+      paper-icon-button {
+        color: #2196F3;
+        border-radius: 100%;
+        pointer-events: auto;  /* TODO(wchargin): why? */
+        width: 32px;
+        height: 32px;
+        padding: 4px;
+      }
+      paper-icon-button[selected] {
+        background: var(--tb-ui-light-accent);
+      }
+    </style>
+  </template>
+  <script>
+    "use strict";
+
+    import {Canceller} from "../tf-backend/canceller";
+    import {getRouter} from "../tf-backend/router";
+    import {runsColorScale} from "../tf-color-scale/colorScale";
+
+    Polymer({
+      is: "tf-distribution-loader",
+      properties: {
+        run: String,
+        tag: String,
+        xType: String,
+
+        _colorScale: {
+          type: Object,
+          value: () => ({scale: runsColorScale}),
+          readOnly: true,
+        },
+        _runColor: {
+          type: String,
+          computed: '_computeRunColor(run)',
+        },
+        _expanded: {
+          type: Boolean,
+          value: false,
+          reflectToAttribute: true,  // for CSS
+        },
+
+        requestManager: Object,
+        _canceller: {
+          type: Object,
+          value: () => new Canceller(),
+        },
+      },
+
+      observers: ['reload(run, tag)'],
+
+      _computeRunColor(run) {
+        return this._colorScale.scale(run);
+      },
+
+      attached() {
+        this._attached = true;
+        this.reload();
+      },
+      reload() {
+        if (!this._attached) {
+          return;
+        }
+        this._canceller.cancelAll();
+        const router = getRouter();
+        const url = router.pluginRunTagRoute("distributions", "/distributions")(
+          this.tag, this.run);
+        const updateData = this._canceller.cancellable(result => {
+          if (result.cancelled) {
+            return;
+          }
+          const backendData = result.value;
+          const data = backendData.map(datum => {
+            // `vz-distribution-chart` wants each datum as an array with
+            // extra `wall_time` and `step` properties.
+            const [wall_time, step, bins] = datum;
+            bins.wall_time = new Date(wall_time * 1000);
+            bins.step = step;
+            return bins;
+          });
+          this.$$('vz-distribution-chart').setVisibleSeries([this.run]);
+          this.$$('vz-distribution-chart').setSeriesData(this.run, data);
+        });
+        this.requestManager.request(url).then(updateData);
+      },
+      redraw() {
```

Comment:
Nit: consistently have blank line between these methods. Docs would be nice too.